languages/ruby/exercises/concept/numbers/.meta/example.rb

```diff
@@ -0,0 +1,32 @@
+class AssemblyLine
+
+  CARS_PER_HOUR = 221
+  MINUTES_IN_HOUR = 60
+  # Success rates
+  ONE_TO_FOUR_RATE = 1.00
+  FIVE_TO_EIGHT_RATE = 0.90
+  NINE_RATE = 0.80
+  TEN_RATE = 0.77
+
+  def self.production_rate_per_hour(speed)
+    CARS_PER_HOUR * speed * success_rate(speed)
+  end
+
+  def self.working_items_per_minute(speed)
+    (production_rate_per_hour(speed) / MINUTES_IN_HOUR).to_i
+  end
+
+  private
```

Comment:
Have we covered method visibility yet?  Does this `private` belong in this exercise?

Reply:
It's an example, so no change required.

Reply:
> It's an example, so no change required.

That's not entirely correct, as the new example files are supposed to be the canonical solution in v3, and no longer just a proof of concept. So the question is correct!

Reply:
Good point, I'll remove that